frontend/src/index.jade

```diff
@@ -109,7 +109,10 @@ html
                     right: (((user.commits.length-j-1)/user.commits.length) * 100) + '%'\
                   }"
                 )
-              .summary-chart__contrib(v-bind:title="'total contribution: '+user.totalCommits")
+              .summary-chart__contrib(
+                 v-bind:title=
+                 "'Total contribution from ' + minDate + ' to ' + maxDate + ': ' + user.totalCommits + ' lines'"
```

Comment:
for line wrap, it should receive twice the indentation, hence 4 spaces from previous line.

Reply:
indentation has been updated